benchmarks/objectExtractionRunningTime.py

```diff
@@ -161,36 +242,47 @@ def _computeFeatures(self, t_ind, t):
             rawVol.squeeze().astype(np.float32), labelVol.squeeze().astype(np.uint32), features, ignoreLabel=0
         )
 
-        # Cleanup results (as done in vigra_objfeats)
```

Comment:
Likewise, this points to more detailed explanations for why the code is the way it is